### ISSUE TITLE
istio-1.1.0.snapshot.1-win.zip missing bookinfo-gateway.yaml

### ISSUE THREAD

**Issue by javajon** [istio/istio#9089]
**Describe the bug**
The istio-1.1.0.snapshot.1-win.zip build artifact has a `samples/bookinfo/networking` directory, **but is empty**. The artifact istio-1.1.0.snapshot.1-linux.tar.gz does have a matching network directory, and that one is full of the expected files.  Previous releases of istio zips for Windows appear to have files in the networking directory.  For instance istio-1.0.2-win.zip has files so defect appears to have been introduced with this recent, specific istio-1.1.0.snapshot.1 release.

**Expected behavior**
`kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml`  should work without error.

**Steps to reproduce the bug**
Expand istio-1.1.0.snapshot.1-win.zip
cd to istio-1.1.0.snapshot.1
```
kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
error: the path "samples/bookinfo/networking/bookinfo-gateway.yaml" does not exist

ll samples/bookinfo/networking/
total 0
drwxr-xr-x 1 User 197608 0 Sep 26 15:34 ./
drwxr-xr-x 1 User 197608 0 Sep 26 15:34 ../
```

**Version**
istioctl 1.1.0.snapshot.1
kubectl 1.12.0

**Installation**
Expand istio-1.1.0.snapshot.1-win.zip to directory

**Environment**
none, Minikube

**Cluster state**
Minikube running and `kubectl apply -f samples/bookinfo/platform/kube/bookin
fo.yaml` worked fine.

**Comment by javajon** [comment]
I just downloaded istio-1.1.0.snapshot.1-win.zip from the same location again and now the zip file download size is 15,310,410 bytes.  When I reported this defect the download zip was was smaller at  15,262,506 bytes.  It appears the file with the same version changed at some point between now and then.  Seems dangerous to have a process that can change the contents of published artifacts without changing the version.

Bottom line it appears the problem is fixed and the files are now present.
